Hippo.Core/Domain/Account.cs

```diff
@@ -42,8 +42,8 @@ public Account()
         public int? OwnerId { get; set; }
         public User Owner { get; set; }
 
-        public int ClusterId { get; set; }
         [Required]

```

Comment:
Do you want this required on the int, or the Cluster below?

Reply:
It has same impact as far as EF is concerned. However, having it on the navigation property is problematic when doing explicit validations in AccountSyncService, because it's setting just the int and not the navigation property.